app/src/main/java/org/stepik/android/view/lesson/ui/delegate/LessonInfoTooltipDelegate.kt

```diff
@@ -96,6 +103,26 @@ class LessonInfoTooltipDelegate(
         }
     }
 
+    // 'StepProgress.score' points out of 'StepProgress.cost' (for step)
+    private fun AppCompatTextView.setItem(
```

Comment:
`setItem` methods are too similar, it will be better to exclude common parts to avoid code duplication